content/faq/editing/advanced-editing/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: How can I do more advanced editing in OpenStreetMap?
+description: "Tutorial for editing OpenStreetMap with more advanced tools like ID"
```

Comment:
It's better to focus this page on two simple things:
1. Teach new users about OSM.
2. Recommend other editors (focus on mobile ones!). And make a link to another FAQ page "How to fix map errors using web ID editor?", it may have different recommendations, including how to copy a link to the place from OM by clicking on coordinates (there is an osm link there too), and other details.

Reply:
The problem is, that there are no mobile editors that are good for beginners and can do "advanced" edits (see https://github.com/organicmaps/organicmaps/pull/7748#issuecomment-2035347759). So ID on a desktop PC is the only good option.

Reply:
What "advanced" cases for new users, in your opinion, can't be handled with existing mobile editors?

Forcing all mobile users to "find and use desktop to edit the map" doesn't look like a good idea. Most new users won't need really "advanced" stuff, from what we see, users need something simpler, that just can't be added/edited yet in OM.
At least a list of user options is needed for mobiles. I would say, that Vespucci and Go! Map can cover most of the cases.

Reply:
I can't say anything about Go! Map, as it's ios only, but Vespucci is rather complicated to use. But mentioning it any way is probably a good idea, as it is the only android editor that can do "advanced" stuff like adding a new road.

Osm Go! and Every Door can only edit POIs and with StreetComplete you can only answer simple quests. So with "advanced" I meant doing more than just editing POIs. So e.g. adding a new road or a building shape.

Have you tried out mobile editors so far? And if yes, how was your experience?

Reply:
"Go! Map" is good enough for mobile. Using it to draw ways and fix areas rarely.

Then the text in the app should be clearer: "Simple map editing apps like Every Door or StreetComplete, or more advanced Vespucci" for Android and "Every Door app for simple map edits, and more advanced Go! Map"